### PR TITLE
[env/github] fix hide glow from Auditorium v1 iframe + hide glow from reactions as well

### PR DESCRIPTION
**Pull request by whoaitsaimz**
Fix build errors introduced by https://github.com/sparkletown/sparkle/pull/1751 as per https://github.com/sparkletown/sparkle/pull/1751#issuecomment-876011919

closes sparkletown/internal-sparkle-issues#817

### CODE REVIEW

**Comment by 0xdevalias** [comment]
@whoaitsaimz This PR has conflicts, presumably because you didn't branch off an updated `env/github` again, but just pushed your old branch with more commits.

image

File: src/scss/customOverrides.scss [hunk 1]
```diff
@@ -10,3 +11,20 @@
...
 }
+
+// hide glow from Auditorium v1 iframe
+.audience-container {
+  .audience-overlay {
+    .video-container {
+      .reaction-container {
+        box-shadow: none;
+      }
+      .video {
```
Comment (0xdevalias) [inline src/scss/customOverrides.scss:22]: Spacing between sibling blocks for readability:
```suggestion
      }

      .video {
```

**Comment by 0xdevalias** [comment]
> Your old code had `border: 1px black;`, the new code doesn't. Is that an explicit choice to remove it? Or an error of omission?
> 
> image

Looking at https://github.com/sparkletown/sparkle/pull/1753/commits/f4d7be77d861b5953d86b87add53e0691d6675ee i'm guessing it was intended.

**Comment by whoaitsaimz** [comment]
> > Your old code had `border: 1px black;`, the new code doesn't. Is that an explicit choice to remove it? Or an error of omission?

Yes intended b/c we don't need to override it.  It's part of the original code I copied. I took it out so it's really only doing what's necessary in the override. With a black background for the page, doesn't make much difference here either way.

**Comment by 0xdevalias** [comment]
> Yes intended b/c we don't need to override it. It's part of the original code I copied. I took it out so it's really only doing what's necessary in the override. With a black background for the page, doesn't make much difference here either way.

Cool, makes sense.

**Review by 0xdevalias — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**